### PR TITLE
Fix test_fbd.py flaky result

### PR DESCRIPTION
**Pull request by wsycqyz**
### Description of PR

Summary:
Fixes test_fbd.py flaky result

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305

#### What is the motivation for this PR?
The result of test_bfd.py is flaky.

#### How did you do it?
Increase sleep time before check bfd status.

#### How did you verify/test it?
Run 300 time and 100% pass on Cisco platform

### CODE REVIEW

**Comment by wsycqyz** [comment]
202305 branch will be in another PR.

File: tests/bfd/test_bfd.py [hunk 1]
```diff
@@ -304,7 +314,9 @@ def create_bfd_sessions_multihop(ptfhost, duthost, loopback_addr, ptf_intf, neig
     ptfhost.template(src='templates/bfd_responder.conf.j2', dest='/etc/supervisor/conf.d/bfd_responder.conf')
     ptfhost.command('supervisorctl reread')
     ptfhost.command('supervisorctl update')
-    ptfhost.command('supervisorctl start bfd_responder')
+    ptfhost.command('supervisorctl restart bfd_responder')
```
Comment (XuChen-MSFT) [inline tests/bfd/test_bfd.py:317]: for first run, wondering whether "restart" still work?

Comment (wsycqyz) [reply tests/bfd/test_bfd.py:317]: Well tested, restart = start if not started.

**Review by XuChen-MSFT — APPROVED** [review]
LGTM

**Pull request merged**